src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/Mocks/IProjectTreeCustomizablePropertyContextFactory.cs

```diff
@@ -4,34 +4,24 @@
 
 using Moq;
 
-#nullable disable
-
 namespace Microsoft.VisualStudio.ProjectSystem.VS
 {
     internal static class IProjectTreeCustomizablePropertyContextFactory
     {
-        public static IProjectTreeCustomizablePropertyContext Create()
-        {
-            return Mock.Of<IProjectTreeCustomizablePropertyContext>();
-        }
-
         public static IProjectTreeCustomizablePropertyContext Implement(
-            string itemName = null,
-            string itemType = null,
+            string? itemName = null,
+            string? itemType = null,
             bool isFolder = false,
             ProjectTreeFlags flags = default,
-            IImmutableDictionary<string, string> metadata = null)
+            IImmutableDictionary<string, string>? metadata = null)
         {
             var mock = new Mock<IProjectTreeCustomizablePropertyContext>();
             mock.Setup(x => x.ItemName).Returns(itemName ?? string.Empty);
-            mock.Setup(x => x.ItemType).Returns(itemType);
+            mock.Setup<string?>(x => x.ItemType).Returns(itemType);
```

Comment:
This happens a fair bit in our repo. The interface which defines `ItemType` is not in our repo and is not annotated. We know it can be `null`, so have to add the type explicitly here to avoid warnings in `Returns`.

Reply:
I'm getting an IDE0001 "Name can be simplified" in the error list for another spot where `<string?>` is specified like this, for this reason. Do you not get that here? Can you add a `#pragma disable` perhaps?

Reply:
On the changes in this PR? Can you give a file and line number?

Reply:
No, from a previous PR presumably. Its here: https://github.com/dotnet/project-system/blob/master/src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/InterceptingPropertyValueProviderBase.cs#L29

Removing the `<string?>` brings back the nullable warning, as you would expect.

This is using VS master, 29213.130

Reply:
I don't see this in 16.2.2 but I do in newer builds. Filed https://github.com/dotnet/roslyn/issues/37980.